simul/monitor/monitor_test.go

```diff
@@ -108,8 +108,10 @@ func setupMonitorStringTest(t *testing.T, testCount int) (*Monitor, *Stats) {
 	m1 := testStringInput{"server1": "crazyStrings"}
 	m2 := testStringInput{"server2": ""}
 	m3 := testStringInput{"server3": "123456789"}
+	m4 := testStringInput{"server3": "crazyString098765432"}
+	m5 := testStringInput{"server3": "456712309crazyString"}
```

Comment:
This is where you put the new input in, which exercises the new desired behaviour of the code under test. Good.

But I do not see the place where you say, "the output from printing the monitor is X, and I've confirmed that the strings I expected to be in there are there".